wikimedia/wikipedia-ios
# Show character count on ArticleDescriptionViewController at all times

**Phabricator:** https://phabricator.wikimedia.org/T332320

### Notes
* With this PR, we are showing the character count on the article description at all times, so the editor has the character limit indication before typing his text

### Test Steps
1. Go to an article, tap add or edit description
2. The character count should always be displayed
